### ISSUE TITLE
Model/View: Node.getText()

### ISSUE THREAD

**Issue by scofalik** [ckeditor/ckeditor5-engine#352]
I missed a simple method, both in Model and View, that would just return the text contents of given node. This should work on a part of a tree, so if an element has elements which has text, it should concat all the textes together.

**Comment by Reinmar** [comment]
Hm... hm... hm... In DOM there are two methods. One which just concatenates strings. Second which format the string, so e.g. blocks are separated by new lines. The latter doesn't make sense in our case at all I guess, but where would you use the former (other than tests)? I can think of "find text" functionality as a usecase. Do we have more?

**Comment by scofalik** [comment]
TBH, I have this as a note without an example :). Probably in tests/tests utils. Maybe in some attribute<->inline element converters.

**Comment by pjasiun** [comment]
But, since it pretty simple to implement using the `Walker` do we really need such method for only few cases?
```
let text = '';
for( values of Range.from( element ) ) {
    if ( values.type == 'TEXT' )
        text += values.item.text;
}
```

**Comment by Reinmar** [comment]
Oh, I think we must have this method. Debugging without it is a nightmare. You cannot check anything about the elements you have in your code. Ability to retrieve text will be the simplest way to check where the heck am I.

**Comment by pjasiun** [comment]
Shouldn't we have a testing tool to display whole content of the element, not only text inside?

**Comment by Reinmar** [comment]
We may and most likely we will have (actually - we do already). But this is such a basic method that it's no harm in implementing it. And it is much easier to debug live with the console if the method is on the element that you have than using an imported tool which is available under transpiled name.

**Comment by Reinmar** [comment]
BTW. @pjasiun implementation won't work on detached elements ;/.

**Comment by Reinmar** [comment]
Implemented in #368.

**Comment by Reinmar** [comment]
This issue covered only the model. If this method will be needed in the view, then we can implement it there.
